src/ja/2017-03/01/01.md

```diff
@@ -0,0 +1,24 @@
+---
+title:  今週のテーマ
+date:   24/06/2017
+---
+
+### 今週のテーマ
```

Comment:
@imasaru With the release of iOS and design principles, we are omitting putting the title as header within the content of the lesson, check out `en` or any other language for `2017-03`